tooling/helpers/evaluate_helper.py

```diff
@@ -0,0 +1,96 @@
+from dataclasses import asdict
+import json
+from helpers.data_helper import BenchmarkResult, RestEvalResult, RestResult, WebsocketEvalResult, WebsocketResult, WebsocketResultBlock
+from helpers.WER_helper import TRANSFORM_DEFAULT
+from helpers.file_helper import (
+    get_file_id,
+    load_export_file_list
+)
+import jiwer
+import pandas
+from Levenshtein import distance as lev
+
+def load_benchmark_result_from_file(filepath: str) -> BenchmarkResult | None:
+    try:
+        with open(filepath,"r") as filehandle:
+            content = json.load(filehandle)
+        return BenchmarkResult(**content)
+    except Exception as e:
+        print(f"Could not read file {filepath} due to an error: {e}")
+        return None
+
+def eval_export_dir() -> pandas.DataFrame:
+    export_files = load_export_file_list()
+    # Axes type = list type -> dont listen to type errors if present
+    rest_df = pandas.DataFrame(columns=['file_id','duration','wer'])
+    websocket_df = pandas.DataFrame(columns=['file_id','duration','wer', 'average_levenshtein_distance'])
+    for filepath in export_files:
+        if (benchmark_res := load_benchmark_result_from_file(filepath)) is None:
+            continue
+        
+        if benchmark_res.expected_transcription is None:
+            continue
+
+        if benchmark_res.rest is not None: 
+            if benchmark_res.rest.faulty:
+                rest_df.loc[len(rest_df)] = {'file_id': get_file_id(filepath), 'duration': benchmark_res.rest.duration, 'wer': None}
```

Comment:
overall I think in df this wouldn't be best practice
The larger the dataset gets the more ineffiction updating dataframes row by row becomes.

Usually you would operate in arrays and then create a DataFrame at once from that.
So I am sure for the reason here in generell